yaksh/views.py

```diff
@@ -3155,3 +3155,18 @@ def get_course_modules(request, course_id):
     modules = course.get_learning_modules()
     context = {"modules": modules, "is_modules": True, "course": course}
     return my_render_to_response(request, 'yaksh/course_detail.html', context)
+
+@login_required
+@email_verified
+def download_course_yaml(request, course_id):
+    course = Course.objects.get(id=course_id)
```

Comment:
Please add the condition to check if the user is moderator and course creator or teacher

Reply:
fixed in subsequent commits